src/main/java/com/epam/test_generator/services/EmailServiceStub.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.test_generator.services;
+
+import com.epam.test_generator.controllers.user.UserDTOsTransformer;
+import com.epam.test_generator.controllers.user.response.UserDTO;
+import com.epam.test_generator.entities.Token;
+import com.epam.test_generator.entities.User;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+@Component
+@Profile("integration-tests")
+public class EmailServiceStub extends EmailService {
```

Comment:
We want to have one interface and two implementations since it is natural design pattern for spring.
Motivation: https://youtu.be/BmBr5diz8WA?t=1670